tests-legacy/Unit/Controller/Admin/AdminTabsControllerTest.php

```diff
@@ -105,8 +105,9 @@ public function setupDatabaseMock($mock = null)
                 if ($subject instanceof DbQuery) {
                     $builtQuery = $subject->build();
 
-                    // It should select modules
-                    return strpos($builtQuery, 'module') !== false;
+                    // It should select modules and test if hook is active
+                    return strpos($builtQuery, 'module') !== false ||
```

Comment:
I'm not sure I understand the reason for this change 🤔

Reply:
The reason is that somewhere the function I modified is called `getHookStatusByName()`, and I have to whitelist it here so that the test my pass.